### ISSUE TITLE
Release tag parity with Sway

### ISSUE THREAD

**Issue by nboughton** [fluix-dev/sway-borders#35]
It looks like there hasn't been a stable release tag since 1.4, would it be feasible to maintain release tag parity with Sway so we can have a stable AUR package and not have to rely on wlroots-git?

**Comment by fluix-dev** [comment]
Keeping parity with Sway releases was actually the intention, as per the README:

> Releases will follow Sway's.

However, I just haven't been too happy with this fork. It... *works* (and I use it daily of course), but there's still some bugs and my intention is to redo most of the work in a way that also allows new features. When that will come I'm not sure, and it would be a breaking change, so maybe tagging a release wouldn't be a bad idea?

I'd like to hear from other users before making a decision though, so please leave your comments below.

**Comment by kushraj** [comment]
definitely, it will be nice to have shadow/other additions which you can use selectively based on unique rules.

One of the obvious gimmick is drop shadow of tiled applications going over taskbar which is not visually pleasing at all.

**Comment by fluix-dev** [comment]
> definitely, it will be nice to have shadow/other additions which you can use selectively based on unique rules.

I hope I can work on it soon! What are your thoughts about releases?

> One of the obvious gimmick is drop shadow of tiled applications going over taskbar which is not visually pleasing at all.

The current border images are drawn on the same layer as the window itself which I think makes sense. Take a look at if your taskbar has a way to draw on a different layer. For example, Waybar can have `"layer": "top"` set to draw *above* and thus your shadows (and window) will indeed be lower.

**Comment by kushraj** [comment]
> Waybar can have "layer": "top" set to draw above

thanks I wasn't aware of that

> What are your thoughts about releases?

you should take your time and go with your thoughts, I will prefer if we can see a good enough implementation of your ideal fork of sway

**Comment by wattahay** [comment]
User thoughts on releases: I would like tag release parity for Fedora.

At this time, Fedora 35 uses Sway 1.6.

The sway-borders master is too new for the dependencies, and 1.4 is too old for the dependencies, with no releases in between.
